src/main/java/seedu/duke/model/itemlist/ItemList.java

```diff
@@ -53,20 +51,12 @@ public ItemList() {
      *
      * @param item Item to be added to the item list.
      */
-    public void addItem(T item) throws DukeException {
+    public void addItem(T item) {
         items.add(item);
         Ui.dukePrint(Messages.MESSAGE_ADD_TASK + item.toString() + Messages.MESSAGE_STATUS_FIRST
                 + items.size() + Messages.MESSAGE_STATUS_LAST);
     }
 
-
-    /**
-     * Adds an item into the list.
-     *
-     * @param description the description of the item
-     */
-    public abstract void addTodo(String description);
-
```

Comment:
I think I also changed this in my other PR might have merge conflicts here later. Oops.

Reply:
I renamed mine to `addTaskFromString` and I put it in `TaskList` instead.

Reply:
Okay thanks!